asmcomp/selectgen.ml

```diff
@@ -366,7 +368,7 @@ method effects_of exp =
     let from_op =
       match op with
       | Capply _ | Cextcall _ | Copaque -> EC.arbitrary
-      | Calloc -> EC.none
+      | Calloc | Cpoll -> EC.none
```

Comment:
This line looks a bit weird. In the middle_end, `Ppoll` has arbitrary effects. In the backend, `Ipoll` is not pure. Here, in Cmm only, `Cpoll` is pure. I think it should be moved next to `Copaque` for the same reason as in the middle-end.